### PR TITLE
DOCS-9479-db-studio-editorial-kt

### PR DESCRIPTION
**Pull request by kevintroller**
DB 1.9 database-connector-studio.adoc editorial

### CODE REVIEW

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Database Connector:
+
+* *Database > On Table Row* +
+Initiates a flow by selecting from a table at a regular interval and generates one message per obtained row
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: Are there two subject verbs here--initiates and generates--or two object verbs here--selecting and generating?
Just double-checking that this sentence uses the intended verb forms for the first option.

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an *On Table Row* source, follow these steps:
+
+. In the *Mule Palette* view, select *Database > On Table Row*.
+. Drag *On Table Row* to the Studio canvas.
+. On the *On Table Row* configuration screen, optionally change the value of the *Display Name* field.
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: "In", I think....
Dang it, I still need to publish something about prepositions to use with GUI elements...Thanks for the reminder!

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. On the *Database Config* window, for *Connection* select any of the connection types to provide to this configuration:
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: ditto

Comment (kbostrom) [reply db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: Again, I would like to see a clearer wording for navigation to a lower-level GUI control, in this case, something like either of...:
```suggestion
. In *Connection* [section? box? menu? list?...] of the *Database Config* window, select any of the connection types to provide to this configuration:
```
OR
```suggestion
. In *Database Config* > *Connection*, select any of the connection types to provide to this configuration:
```

Comment (kbostrom) [reply db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: GLOBAL for steps worded this way throughout this set of topics.

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+* *Data Source Reference Connection*
+* *Derby Connection*
+* *Generic Connection*
+* *Microsoft SQL Server Connection*
+* *MySQL Connection*
+* *Oracle Connection*
+
+[start=6]
+. On the *General* tab, specify the database connection information for the connector.
+. On the *Transactions* tab, optionally specify the transaction isolation, or XA transactions when connecting to the database.
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: Does "when connecting to the database" refer only to XA transactions or also to transaction isolation?
The comma confuses me.
"when connecting" also confuses me: do you mean that the audience does this when they are connecting, or do you mean that the isolation and/or transactions occur during database connection?

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+. On the *Advanced* tab, optionally specify connection pooling and reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified database.
+. Click *OK* to close the window.
+. On the *On Table Row* configuration screen, in *Table*, specify the name of the table to select from.
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: ditto prev. about clarifying nav. flow

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Database Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Database Connector* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the input source.
+
+The following screenshot shows the Database Connector operations in the Mule Palette of Anypoint Studio:
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]:
```suggestion
The following screenshot shows the Database Connector operations in the Mule Palette view of Anypoint Studio:
```

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+.Database Connector Operations
+image::database-operations.png[Database Connector Operations]
+
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Database Connector can connect to any database for which a JDBC driver is available. To make it easier to connect to the most popular databases (such as MySQL, Oracle, and Microsoft SQL Server), the connector includes a series of different connection types.
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: "it's best to" is optional. Removing it makes clear that we are actually telling the user to DO this thing.

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+To configure the global element for Database Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *Database Config* window, for *Connection* select any of the connection types to provide to this configuration:
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: ditto re. nav wording

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+* *Data Source Reference Connection*
+* *Derby Connection*
+* *Generic Connection*
+* *Microsoft SQL Server Connection*
+* *MySQL Connection*
+* *Oracle Connection*
+
+[start=4]
+. On the *General* tab, specify the database connection information for the connector.
+. On the *Transactions* tab, optionally specify the transaction isolation, or XA transactions when connecting to the database.
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]: ditto prev. question about wording

File: db/1.9/modules/ROOT/edits/database-connector-studio.adoc [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+. On the *Advanced* tab, optionally specify connection pooling and reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified database.
+. Click *OK*.
+
+The following screenshot shows the Database Connector Global Element Configuration window in Anypoint Studio:
```
Comment (kbostrom) [inline db/1.9/modules/ROOT/edits/database-connector-studio.adoc]:
```suggestion
The following screenshot shows the *Database Connector Global Element Configuration* window in Anypoint Studio:
```

**Review by kbostrom — DISMISSED** [review]
CE per request

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**